dd4hep.spec

```diff
@@ -23,8 +23,7 @@ CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX='%{i}' \
       -DDD4HEP_USE_PYROOT=ON \
       -DCMAKE_CXX_STANDARD=17 \
       -DCMAKE_BUILD_TYPE=Release \
-      -DDD4HEP_USE_TBB=ON \
-      -DCMAKE_PREFIX_PATH=${TBB_ROOT}/cmake/TBB;${CLHEP_ROOT};${XERCES_C_ROOT}"
+      -DCMAKE_PREFIX_PATH=${TBB_ROOT}/lib/cmake/TBB;${CLHEP_ROOT};${XERCES_C_ROOT}"
```

Comment:
@mrodozov , do we still need `${TBB_ROOT}/lib/cmake/TBB` and tbbdeps here?

Reply:
right. we don't. it's configuring without it (I checked)